wiki/Ranking_criteria/en.md

```diff
@@ -42,6 +42,7 @@ Important to understand before using:
 ### Guidelines
 
 - **Creators of the media used in a beatmap should be credited in the beatmap's description.** Per the [Content Usage Guidelines](/wiki/Rules/Content_usage_guidelines), media used in beatmaps should be permitted for use in osu!, and their licensing will likely detail how to credit their creators. Common reasons to omit credits from the description include exceeding difficulty in sourcing, creators' wishes to remain uncredited, and presence of credits in the artist, title, or source fields.
+  - Beatmaps using AI-generated backgrounds should instead credit them as such in the description.
```

Comment:
If we're looking to make this as clear as day, I'd suggest something like this:

```suggestion
  - Beatmaps with AI-generated backgrounds should explicitly mention this in the description. This is to avoid any controversy from creators incorrectly claiming creative input (ie. claiming to have "illustrated" the background themselves).
```

Reply:
I'm keen on making the intent behind the clarification clearer.

Two things I'd change would be `controversy` to something less informal or "direct" as the purpose of crediting these works is also to be informative. Second thing would be changing `ie.` for `e.g.` which seems to be used in the RC when providing examples.

```suggestion
  - Beatmaps with AI-generated backgrounds should explicitly mention this in the description. This is to avoid creators from incorrectly claiming creative input (e.g. claiming to have "illustrated" the background themselves).
```

Reply:
Fair points. I also thing it should be generalised to all imagery, not just backgrounds. How does this phrasing feel?

```suggestion
  - Beatmaps with AI-generated imagery should explicitly attribute it as being AI-generated, not "illustrated" or otherwise. This helps avoid any ambiguity over the creative input / process.
```

Reply:
When it comes to phrasing I'd probably not be the best person to ask haha.

Using "imagery" may blur the line a little, as we aren't referring to the files themselves but the contents of backgrounds/videos. Say for example, a background with AI assets or tweaks would potentially fall there, even if it wasn't fully generated by a model. Is this something we want to account for? 

Otherwise, we could use "media" or a similar variation of that as you pointed out before to refer to files in their entirety.

Will need more eyes on `This helps avoid any ambiguity over the creative input / process.` as I cannot judge how well it fits wording wise. But the idea feels good.

Reply:
Imagery was intended to include storyboard etc.

Reply:
Hmm alright yeah fair. Not opposed to it if this is meant to encompass those kinds of assets.

Reply:
the wording doesn't need to be opinionated or embellished beyond the main point of marking ai content as ai content. i don't get why a point about "creative input" needs to be made in the same breath

```suggestion
  - AI-generated media in beatmaps should explicitly be attributed as being AI-generated.
```

both the original beatmap thread and internal nat thread also seem to partly revolve around not wanting people to say ai images are "illustrated", although if they're credited as "ai-generated" anyway then is such a remark really necessary?

Reply:
It provides reasoning. Something which is sorely missing from the guidelines at the moment.

Reply:
well maybe something like this then

```suggestion
  - AI-generated media in beatmaps should explicitly be attributed as being AI-generated, not illustrated or otherwise. This is to prevent misrepresentation of the creation process involved.
```

basically the last suggestion here but more direct

Reply:
Sure, I can agree on that one.

Reply:
Looks good!